### PR TITLE
Frag cleanup

### PR DESCRIPTION
**Pull request by ulteq**
Fixes: https://github.com/acemod/ACE3/issues/824

Review needed.

### CODE REVIEW

**Comment by ulteq** [comment]
We should also disable the firedEH if the projectile does not inherit from GrenadeBase.

**Comment by NouberNou** [comment]
Why would we disable the firedEH for that? Tons of other things that cause frag inherit from stuff other than GrenadeBase....

File: addons/frag/functions/fnc_addBlackList.sqf [hunk 1]
```diff
@@ -1,4 +0,0 @@
-#include "script_component.hpp"
-private ["_round"];
-_round = _this select 0;
-GVAR(blackList) set[(count GVAR(blackList)), _round];
```
Comment (Glowbal) [inline addons/frag/functions/fnc_addBlackList.sqf]: Are we removing the blacklist? This function might actually be useful to keep around. Though we do need to adjust it to a pushback.

**Comment by ulteq** [comment]
> Tons of other things that cause frag inherit from stuff other than GrenadeBase....

Makes sense.

> Are we removing the blacklist?

No, the blacklist is still in place, but the addBlackList function was never called, so I removed it for now.

**Comment by Glowbal** [comment]
But we could use it for access by third parties. That way we do not have to expose the variable, but only the function. ref: #820

**Comment by ulteq** [comment]
Agreed. I'll revert the function.

**Comment by ulteq** [comment]
@NouberNou, can we merge it?

**Comment by jaynus** [comment]
Lets do #342 in this branch too first, and then merge.

**Comment by ulteq** [comment]
Ok

**Comment by gienkov** [comment]
By just simply looking I assume GBU12 will spawn loads of shrapnels? Will it impact framerate?

**Comment by jaynus** [comment]
Well you shouldn't be dropping 50 GBU's all at the same time, so no :p

**Comment by ulteq** [comment]
> Lets do #342 in this branch too first, and then merge.

Those are done:
* G_40mm_HE
* G_40mm_HEDP
* Sh_82mm_AMOS
* Sh_155mm_AMOS

**Comment by ulteq** [comment]
I will merge this tomorrow, if there are no objections.

**Pull request merged**